app.js

```diff
@@ -0,0 +1,38 @@
+const axios = require("axios");
+const dotenv = require('dotenv');
+dotenv.config();
+const apiURLPrefix = "http://api.openweathermap.org/";
+
+async function PrintWeather(input) {
```

Comment:
Why is this function capitalized?

All the functions need a Docblock.

This method is called `printWeather` but it is also making an API request. This feels unnatural to me.
Lets split this up.
Get the weatherData in one method, add a method that builds the output, then handle the output where that is used.

Happy to talk about that if its confusing.

Reply:
I think I got it, let me know what you think.